source/developers-guide/custom-shopping-world-elements/index.md

```diff
@@ -306,9 +306,23 @@ There are several methods for nearly any kind of form field. Here is a list of a
                 <li><code>helpTitle</code></li>
                 <li><code>helpText</code></li>
                 <li><code>allowBlank</code></li>
+                <li><code>valueField</code></li>
             </ul>
         </td>
-        <td><img src="img/screen_mediafield.jpg" alt="media field" /></td>
+        <td>
+            <img src="img/screen_mediafield.jpg" alt="media field" />
+            <br /><br />
+            <p>With the <code>valueField</code>, you can control which property of the MediaModel you want to return as a return value</p>
```

Comment:
The <code>valueField</code> allows you to control which property of the MediaModel is returned as value